php-nginx/README.md

```diff
@@ -19,6 +19,15 @@ docker-compose build php56_nginx
 docker-compose push php56_nginx
 ```
 
+## About
+
+This is a Docker image for PHP (using the php-fpm SAPI) and Nginx. It uses, by
+default, our recommended default configuration, including:
+
+* A default HTTPS only website, with HTTP redirecting to HTTPS
```

Comment:
This should be considered both for environments that don't have an SSL LB terminator, and those where the security requirements ask that inter-container traffic is TLS encrypted. Getting Vault to manage the TLS certs will help make that an easy reality for any project that choses to once the groundwork is done.